code/controllers/subsystem/SSticker.dm

```diff
@@ -405,29 +405,28 @@ SUBSYSTEM_DEF(ticker)
 					flick("intro_nuke", cinematic)
 					sleep(35)
 					flick("station_explode_fade_red", cinematic)
+					stop_delta_alarm()
```

Comment:
Why not just move `stop_delta_alarm()` somewhere before this switch if it'll be in every branch anyway?

Reply:
because of timings, sounds ridiculous but the 3.5s sleep throws the timing off in a way that makes it seem janky

Reply:
Can't you put it at the end of the switch then? Still in the else scope

Reply:
oh yeah, done @farie82